src/main/java/org/folio/consortia/controller/SelfController.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.consortia.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.consortia.domain.dto.UserTenantCollection;
+import org.folio.consortia.rest.resource.SelfApi;
+import org.folio.consortia.service.UserTenantService;
+import org.folio.spring.FolioExecutionContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/consortia/{consortiumId}")
+@RequiredArgsConstructor
+public class SelfController implements SelfApi {
+
+  @Autowired
```

Comment:
@Autowired can be ommited as in TenantController
https://github.com/folio-org/mod-consortia/pull/38/files#diff-e317befe897b391a97ea21cc29efd0c64d86656e98c380720d528d74943a198dL24